src/Catalog.GraphQL/Models/ProductDto.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Catalog.GraphQL.Models;
+public class ProductDto
+{
+    public int Id { get; set; }
+    public string Name { get; set; }
+
+    public string Description { get; set; }
+
+    public int Image { get; set; }
+
+    public int CatalogId { get; set; }
+    
+    public decimal Price { get; set; }
+
+    public uint Amount { get; set; }
+
+    public static ProductDto FromProduct(CatalogService.Domain.Entities.Product product)
+    {
+        return new ProductDto()
```

Comment:
Did you considered introducing some mapper library?

Reply:
Yep, we could use Mapper or something. But that is a demonstration example. If it is really need I could add it.

Reply:
In this case, I have switched to the Application level with all types and mappings.